examples/basic/main.tf

```diff
@@ -10,22 +10,24 @@ module "resource_group" {
   existing_resource_group_name = var.resource_group
 }
 
-########################################################################################################################
-# COS
-########################################################################################################################
+##############################################################################
+# Cloud Monitoring
+##############################################################################
 
 #
 # Developer tips:
```

Comment:
this can be removed its only needed in the initial template as a guide for developers creating a module example